parastell/nwl_utils.py

```diff
@@ -274,40 +305,61 @@ def nwl_plot(
     """
     tor_ext = np.deg2rad(tor_ext)
     pol_ext = np.deg2rad(pol_ext)
-    
+
     coords = extract_coords(source_file)
 
     if num_crossings is not None:
         coords = coords[0:num_crossings]
-    
-    # Load plasma equilibrium data
+
+    if num_threads is None:
+        num_threads = cpu_count()
```

Comment:
That's agressive - can we say `cpu_count()/2` or something as the default to avoid taking all the resources?

Reply:
Default could also be 1??